.env

```diff
@@ -1 +0,0 @@
-NODE_PATH=src
```

Comment:
We are using absolute imports, if we decide not to use .env then we need to add aliases in webpack.config. Please add those changes so we keep importing like `components/Layouts` or `components`

Reply:
Maybe other configuration like this https://medium.com/@kkomaz/another-solution-to-absolute-path-setup-in-create-react-app-cccb38d8eea8 @hdwilber